bin/build-locales

```
@@ -189,56 +189,47 @@ async.forEachLimit(views, 5, function (view, cb) {
+        var translationsFile = path.resolve(
+            __dirname,
+            '../',
+            localesDir,
+            'scratch-website.' + view + '-l10njson',
+            isoCode + '.json'
+        );
+        fs.readFile(translationsFile, 'utf8', function (err, data) {
+            if (err) {
+                if (err.code === 'ENOENT') {
+                    if (isoCode !== 'en') {
+                        if (defaultLocales.hasOwnProperty(view)) {
                             process.stdout.write('No translations for ' + view + isoCode + ', using english\n');
-                            viewLocales[isoCode] = merge({}, generalLocales[isoCode], defaultLocales[view]);
                         }
-                        return cb();
-                    } else {
-                        return cb(err);
+                        viewLocales[isoCode] = merge({}, generalLocales[isoCode], defaultLocales[view]);
                     }
-                }
-                try {
-                    viewLocales[isoCode] = merge({}, generalLocales[isoCode], JSON.parse(data));
-                } catch (e) {
-                    return cb(e);
-                }
-                cb();
-            });
-        }, function (err) {
-            if (err) process.stdout.write('Error merging translations for view: ' + view + '\n' + err + '\n');
-            var viewTranslations = merge({}, viewLocales, localizedAssetUrls[view]);
-            writeIntlFile(outputDir, view, viewTranslations, function (err) {
-                if (err) {
-                    process.stdout.write('Failed to save: ' + view + '\n');
+                    return cb();
+                } else {
                     return cb(err);
                 }
-            });
+            }
+            try {
+                viewLocales[isoCode] = merge({}, generalLocales[isoCode], JSON.parse(data));
+            } catch (e) {
+                return cb(e);
```

I agree - let's fail hard on this one. It means that data in the scratch_translations repo is bad and probably we should fix it upstream.